client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.html

```diff
@@ -1,41 +1,58 @@
-<form id="vmDeletionForm" name="vmDeletionForm" novalidate class="oui-modal oui-modal_shadow">
-    <div class="oui-modal__header">
-        <button class="oui-icon oui-icon-close_thin oui-modal__close-button" type="button" aria-label="Exit"
-                data-ng-click="$ctrl.cancel()"></button>
-    </div>
-    <div class="oui-modal__body oui-modal__body_no-icon" data-ng-if="$ctrl.loaders.ips">
-        <div class="text-center">
-            <oui-spinner></oui-spinner>
+<form class="oui-modal oui-modal_shadow">
```

Comment:
(advertising for nice projects) This could have been passed to ui-angular component :P